i18n/es/docusaurus-plugin-content-docs/version-V3/resources.md

```diff
@@ -4,16 +4,30 @@ sidebar_position: 8
 
 # Recursos
 
-[¡La versión 2 de Semaphore está lista!](https://medium.com/privacy-scaling-explorations/semaphore-v2-is-live-f263e9372579) - Privacy and Scaling Explorations
+## Artículos
```

Comment:
@vplasencia

If we change something here, we should also change the structure of the English doc (?)

Reply:
I think it would be good to do it like in the Spanish version. What do you think?

Reply:
Yes, I agree. Could you update the English version before merging?

Reply:
Yes, I can do it